automation/WinFormsAutomationApp/WinFormsAutomationApp.csproj

```diff
@@ -40,15 +43,6 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
-  <PropertyGroup>
```

Comment:
I've made the same changes to all of the projects - imported the new settings file that contains _DelaySign_, _SignAssembly_ and _AssemblyOriginatorKeyFile_, and deleted those properties from the project.